samcli/lib/providers/sam_function_provider.py

```diff
@@ -97,16 +97,15 @@ def get(self, name: str) -> Optional[Function]:
             if len(found_fs) > 1:
                 found_fs.sort(key=lambda f0: f0.full_path.lower())
 
-                LOG.warning(
-                    "Multiple functions found with keyword %s! Function %s will be invoked! "
-                    "If it's not the function you are going to invoke,"
-                    "please choose one of them from below:",
-                    name,
-                    found_fs[0].full_path,
+                message = (
+                    f"Multiple functions found with keyword {name}! Function {found_fs[0].full_path} will be "
+                    f"invoked! If it's not the function you are going to invoke, please choose one of them from"
+                    f" below:"
```

Comment:
`name,found_fs[0].full_path,` Do we want to add these to the message?

Reply:
they are already there 
`f"Multiple functions found with keyword <<<{name}!>>>> Function <<<<{found_fs[0].full_path}>>>> will be "`